torch/csrc/generic/Tensor.cpp

```diff
@@ -770,7 +770,6 @@ static bool THPTensor_(_convertToTensorIndexers)(
 
           // Clean up Indexers
           for (auto& idx : indexers) {
-            THIndexTensor_(free)(LIBRARY_STATE idx->cdata);
             Py_DECREF(idx);
```

Comment:
Ugh `indexers` should just become a vector of `THPObjectPtr`s, so we don't have to clean it up in 4 places

Reply:
@zou3519 can you send a follow-up PR making `indexers` be a vector of `THPObjectPtr`

Reply:
Will do